narzedzia/gatling

```diff
@@ -0,0 +1,45 @@
+# Gatling
+
+Gatling to biblioteka do testów wydajnościowych, którą można również stosować do testów funkcjonalnych. Możemy korzystać z całej gamy gotowych protokołów, jak również rozszerzać bibliotekę do własnych potrzeb, używajac języka Scala.
+
+### Strona projektu
+* [https://gatling.io/](https://gatling.io/)
+
+### Zestaw przygotowany przez autorów
+* [https://gatling.io/documentation/](https://gatling.io/documentation/)
+
+### Materiały do nauki Gatlinga
+* [https://www.baeldung.com/introduction-to-gatling](https://www.baeldung.com/introduction-to-gatling)
+* [https://www.swtestacademy.com/introduction-gatling/](https://www.swtestacademy.com/introduction-gatling/)
+
+### Materiały do nauki Scali
```

Comment:
Scala jest językiem programowania. Nie powinna znajdować sie w folderze "narzędzia".
Tymm bardziej materiały do języka, który można wykorzystać do wielu narzędzi nie powinny znajdować się tylko przy jednym narzędziu.
Do usunięcia lub wyniesienia jako podstrona języka skala w fodlerze skupiającym języki przydatne testerom. Na ten pomysł nie mam jeszcze pomysłu gdzie, możemy wspólnie pomyśleć :)

Reply:
Z jednej strony tak, ale z drugiej strony - przy Selenium to rzeczywiście jest wybór, natomiast używanie Gatlinka już się wiąże ze Scalą. Dlatego bym to mimo wszystko zostawił, a jak będzie osobna sekcja na języki to wtedy zalinkował do niej

Reply:
Fair enough, dodam #TODO ze zstosowną informacją :)